src/main/java/com/immunopass/constants/RequestHeader.java

```diff
@@ -0,0 +1,9 @@
+package com.immunopass.constants;
+
+import lombok.experimental.UtilityClass;
+
+@UtilityClass
+public class RequestHeader {
+
+    public static String ACCESS_TOKEN = "at";
```

Comment:
What's this token?

Reply:
key corresponding to header which will send accessToken

Reply:
Is "at" a standard header for jwt? If no, then please check what's the standard header for sending the auth token.

Reply:
Not the standard.. couldn't find anything standard name for it. We can change it later on